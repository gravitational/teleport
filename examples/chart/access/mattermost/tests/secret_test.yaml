suite: Test secret
templates:
  - secret.yaml
tests:
  - it: should contain the token
    set:
      mattermost:
        token: mymattermosttoken
    asserts:
      - matchSnapshot: {}

  - it: should not exist when using external secret
    set:
      mattermost:
        tokenFromSecret: my-mattermost-secret
    asserts:
      - hasDocuments:
          count: 0
