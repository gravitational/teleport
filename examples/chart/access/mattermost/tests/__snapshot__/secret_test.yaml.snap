should contain the token:
  1: |
    apiVersion: v1
    data:
      mattermostToken: bXltYXR0ZXJtb3N0dG9rZW4=
    kind: Secret
    metadata:
      name: RELEASE-NAME-teleport-plugin-mattermost-secret
    type: Opaque
